pkg/agent/openflow/pipeline_windows.go

```diff
@@ -311,3 +311,54 @@ func (c *client) l3FwdFlowToRemoteViaRouting(localGatewayMAC net.HardwareAddr, r
 	}
 	return []binding.Flow{c.l3FwdFlowToRemoteViaGW(localGatewayMAC, *peerPodCIDR, category)}
 }
+
+func (c *client) arpResponderClusterIPFlows(svcIP net.IP, gwMAC net.HardwareAddr, category cookie.Category) (flows []binding.Flow) {
+	flows = append(flows, c.pipeline[arpResponderTable].BuildFlow(priorityHigh).MatchProtocol(binding.ProtocolARP).
+		MatchARPTpa(svcIP).MatchARPOp(arpOpRequest).
+		Action().LoadARPOperation(arpOpReply).
+		Action().Move(binding.NxmFieldSrcMAC, binding.NxmFieldDstMAC).
+		Action().SetSrcMAC(gwMAC).
+		Action().Move(binding.NxmFieldARPSha, binding.NxmFieldARPTha).
+		Action().SetARPSha(gwMAC).
+		Action().Move(binding.NxmFieldARPSpa, binding.NxmFieldARPTpa).
+		Action().SetARPSpa(svcIP).
+		Action().OutputInPort().
+		Cookie(c.cookieAllocator.Request(category).Raw()).
+		Done())
+	return flows
+}
+
+func (c *client) clusterIPHostNetworkFlows(gwIP net.IP, nodeIP net.IP, category cookie.Category) (flows []binding.Flow) {
+	flows = append(flows,
+		c.pipeline[l3ForwardingTable].BuildFlow(priorityLow).MatchProtocol(binding.ProtocolIP).
```

Comment:
Could you add some comments to explain the flows?

Reply:
Added some comments.